docs/_posts/examples/3400-01-31-animate-a-line.html

```diff
@@ -0,0 +1,128 @@
+--- layout: example category: example title: Animate a line description: Animate a line by updating a GeoJSON source on each frame. tags: - layers - sources ---
+<style>
+button {
+  position: absolute;
+  margin: 20px;
+}
+
+#play::after {
+  content: 'Stop';
+}
+
+#play.pause::after {
+  content: 'Play';
+}
+</style>
+
+<div id='map'></div>
+<button id='play' class='pause'></button>
+
+<script>
+mapboxgl.accessToken = 'pk.eyJ1IjoieXVuamllbGkiLCJhIjoiY2lxdmV5MG5rMDAxNmZta3FlNGhyMmpicSJ9.CTEQgAyZGROcpJouZuzJyA';
```

Comment:
you don't need an access token here. a token is automatically added by jekyll when it compiles the examples